Apps/Admin/Client/Components/Delegation/DependentTable.razor.cs

```diff
@@ -51,7 +51,7 @@ public partial class DependentTable : FluxorComponent
 
         private bool InEditMode => this.DelegationState.Value.InEditMode;
 
-        private IEnumerable<DependentRow> Rows => new List<DependentInfo> { this.Data }.Select(d => new DependentRow(d));
+        private IEnumerable<DependentRow> Rows => new[] { this.Data }.Select(d => new DependentRow(d));
```

Comment:
```suggestion
        private IEnumerable<DependentRow> Rows => [ new DependentRow(this.Data ) ];
```
Isn't this equivalent?

Reply:
Good catch.  I had tried a direct swap out of the List creation with the new collection expression - `[this.Data].Select(d => new DependentRow(d))` - but it wasn't able to interpret the type it should create when the Select() was involved.  I hadn't noticed that the Select() was largely unnecessary so I'll update it per your suggestion.